.github/workflows/build-and-test.yml

```diff
@@ -76,8 +76,6 @@ jobs:
           - '3.5.1'
           - '3.6.3'
           - '4.1.0'
-          - '4.2.0'
```

Comment:
The previous build couldn't handle multiple parameters for 4.2.0 properly. FYI, 4.2.0's parameters are already defined in `include` section and it's the correct way to add multiple parameters.